test/functional/pdo_sqlsrv/MsCommon_mid-refactor.inc

```diff
@@ -139,6 +165,15 @@ class ColumnMeta
         }
         $this->dataType = $dataType;
         $this->options = $options;
+        if (is_null($encType)) {
+            if (isColEncrypted()) {
+                $this->encType = "deterministic";
+            } else {
+                $this->encType = "normal";
+            }
+        } else {
+            $this->encType = $encType;
+        }
```

Comment:
I'm not sure I understand the logic. $encType is null by default
Looks like it's either `deterministic` or `normal`?

Reply:
`$encType` is null by default so when the user does not provide an argument for this ("normal", "deterministic" or "randomized"), the `encType` is set based on whether or not isColEncrypted is true (i.e., when isColEncrypted is true, "deterministic" is the default encType; when isColEncrypted isn't true, "normal" is the encType). If the user provides an argument however, the `encType` is whatever that the user provides.

Reply:
exactly, so if `encType` is null, it's no encryption, correct? 
therefore, you can leave `encType` = null

Reply:
if encType is null that means user is lazy and relies on MsCommon it pick the encType